### PR TITLE
Add option to link to Worldopole

### PR DESCRIPTION
**Pull request by 123FLO321**
Add an option that adds a "Full Stats" button in stats, that links to Wordlopole.

### CODE REVIEW

**Comment by hammydown4325** [comment]
Did u mean to do this on develop? Glennmen said he wouldn't approve PRs not on develop?

**Comment by 123FLO321** [comment]
@iroken222 Yes i did. Changed now

**Comment by davidrhino** [comment]
I would like to see us move away from using the <center> tag in HTML as its been deprecated since html 4.  I plan on cleaning it up in all of our codebase as we go forward but definitely could start here. Instead you can target the switch-container and give it text-align:center;

So basically instead of 
`
<div class="switch-container">
<div><center><a href="<?= $worldopoleUrl ?>">Full Stats</a></center></div>
</div>
`
`
<div class="switch-container full-stats">
      <a href="<?= $worldopoleUrl ?>">Full Stats</a>
</div>
`
Then in the scss files add
`
.full-stats{
text-align:center;
}
`

This is a much more modern way of handling this solution as center is frowned upon in the web community.

**Comment by hammydown4325** [comment]
I would like to see us move away from using the `<center>` tag in HTML as its been deprecated since html 4.  I plan on cleaning it up in all of our codebase as we go forward but definitely could start here. Instead you can target the switch-container and give it text-align:center;

So basically instead of 
```
<div class="switch-container">
<div><center><a href="<?= $worldopoleUrl ?>">Full Stats</a></center></div>
</div>
```

```
<div class="switch-container full-stats">
      <a href="<?= $worldopoleUrl ?>">Full Stats</a>
</div>
```
Then in the scss files add
```
.full-stats{
text-align:center;
}
```

This is a much more modern way of handling this solution as center is frowned upon in the web community.

File: pre-index.php [hunk 1]
```diff
@@ -602,9 +602,15 @@ class="onoffswitch-checkbox"/>
...
     <nav id="stats">
         <div class="switch-container">
-            <!--<div class="switch-container">
-                <div><center><a href="stats">Full Stats</a></center></div>
-            </div>-->
+            <?php 
+            if ($worldopoleUrl != "") {
+                ?>
+                <div class="switch-container">
+                    <div><center><a href="<?= $worldopoleUrl ?>">Full Stats</a></center></div>
```
Comment (peterchabs) [inline pre-index.php]: I feel just the blue hyperlink `Full Stats` is a bit uninviting and easily overlooked, perhaps when enabled could trigger the Worldopole Icon along with it? (Or icon of choice)
Using on my map, no issues.

File: pre-index.php [hunk 2]
```diff
@@ -602,9 +602,15 @@ class="onoffswitch-checkbox"/>
     </nav>
     <nav id="stats">
         <div class="switch-container">
-            <!--<div class="switch-container">
-                <div><center><a href="stats">Full Stats</a></center></div>
-            </div>-->
+            <?php 
+            if ($worldopoleUrl != "") {
```
Comment (hammydown4325) [inline pre-index.php]: Change this to !== to prevent php type conversion

**Review by hammydown4325 — CHANGES_REQUESTED** [review]
Overall very good just add the extra = to prevent the type conversion and its good to merge.

**Pull request merged**